app/src/main/java/io/left/meshim/adapters/MessageListAdapter.java

```diff
@@ -132,6 +155,17 @@ public void onBindViewHolder(MessageViewHolder holder, int position) {
                 } else {
                     holder.mDeliveryStatus.setImageResource(R.drawable.ic_done_all_black_24dp);
                 }
+
+                if(!message.getFilePath().equals("")){
```

Comment:
This looks like duplicate code from above. Why make this an else case? Why not have "if there's a filename, set the file; then if there's a message, set the message?"